### PR TITLE
Create guide about Shinken as a Service

### PR DESCRIPTION
**Pull request by NicolasLM**
Signed-off-by: Nicolas Le Manchet <nicolas.le-manchet@ovh.net>

### CODE REVIEW

File: en/_posts/2016-06-22-getting-started-with-shinken-as-a-service.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+title:  "Getting started with Shinken as a Service"
+categories: shinken
+author: NicolasLM
+lang: en
+---
+
+# Shinken as a Service
+
+Shinken is a monitoring software designed to keep an eye on your infrastructure and alert you if
+anything happens. It is designed to scale to handle large setups.
```
Comment (yadutaf) [inline en/_posts/2016-06-22-getting-started-with-shinken-as-a-service.md]: typo: to scale and* handle...

File: en/_posts/2016-06-22-getting-started-with-shinken-as-a-service.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+As we believe that anyone should be able to monitor their services without investing a lot of time
+and money in deploying monitoring software, we built Shinken as a Service: a simple solution that
+provides you a fully managed dedicated Shinken cluster.
+
+You can use this cluster to monitor your OVH products (dedicated servers, VPS, cloud instances...)
+as well as websites and on-premise systems.
+
+# Getting started
+
+During the lab, anyone can activate a monitoring stack per OVH account.
```
Comment (yadutaf) [inline en/_posts/2016-06-22-getting-started-with-shinken-as-a-service.md]: Does "a" means "any" or "1" in this context?

File: en/_posts/2016-06-22-getting-started-with-shinken-as-a-service.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+layout: post
+title:  "Getting started with Shinken as a Service"
+categories: shinken
+author: NicolasLM
+lang: en
+---
+
+# Shinken as a Service
+
+Shinken is a monitoring software designed to keep an eye on your infrastructure and alert you if
```
Comment (yadutaf) [inline en/_posts/2016-06-22-getting-started-with-shinken-as-a-service.md]: Maybe add a link to the official Shinken website for more informations ?

File: en/_posts/2016-06-22-getting-started-with-shinken-as-a-service.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+cloud infrastructures.
+
+### Alerts
+
+By default a contact called ``admin`` is created when you deploy the stack. Change the default
+email address associated to the contact to receive alerts by email.
+
+### Thruk web interface
+
+Every Shinken cluster comes with a web dashboard showing the state of your infrastructure. The
```
Comment (yadutaf) [inline en/_posts/2016-06-22-getting-started-with-shinken-as-a-service.md]: Maybe add a link to Thruk website for more Informations / Screenshots ?

File: en/_posts/2016-06-22-getting-started-with-shinken-as-a-service.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+### Passive checks
+
+By default your Shinken only uses active checks where Shinken contacts your host or service to
+retrieve its status. You can also enable passive checks where your host pushes checks to your
+Shinken cluster. A common way to do that is through NSCA.
+
+NSCA is a simple protocol to submit host and service checks to a monitoring server. When activating
+it you will be able to choose:
+
+- an encryption method, currently supported modes are 0 (no encryption) and 1 (XOR obfuscation).
```
Comment (yadutaf) [inline en/_posts/2016-06-22-getting-started-with-shinken-as-a-service.md]: I feel bad using "encryption" and "XOR" in the same sentence ;)

File: en/_posts/2016-06-22-getting-started-with-shinken-as-a-service.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+- a key, which should be very long and random. Only used with encryption method 1.
```
Comment (yadutaf) [inline en/_posts/2016-06-22-getting-started-with-shinken-as-a-service.md]: Yeah, like *really* lonk like longer than the longest message to send

**Comment by yadutaf** [comment]
Nice and clear. Is there any specific limitations that we should advertise here ?

**Comment by NicolasLM** [comment]
Thanks for your review, I pushed an updated version.
Currently I have nothing more to add, I will update it if needed after getting feedback from users.

**Comment by yadutaf** [comment]
Merged!

**Pull request merged**